test/HttpClient.test.js

```diff
@@ -26,12 +26,14 @@ describe('HttpClient', function () {
             CLIENT = clients.createHttpClient({
                 url: 'http://www.restify.com'
             });
-            assert.strictEqual(CLIENT.url.protocol, 'http:');
+        });
+        assert.strictEqual(CLIENT.url.protocol, 'http:');
 
+        assert.doesNotThrow(function () {
```

Comment:
What is the behavior of `doesNotThrow` when you don't provide any other arguments? Does it just re-throw this error? Do we need an assert here? If it throws, wont the test fail?

http://chaijs.com/api/assert/#method_doesnotthrow

Reply:
Correct, it just rethrows. In this case you're right, we probably don't need it, but I usually leave it in just to be explicit about what I'm testing (and in some scenarios where behavior may not be obvious because you're depending on default behavior). Don't feel super strongly though, if you'd prefer to remove it.